### PR TITLE
Added Data Prepper 1.4.0 Archives 

### PR DESCRIPTION
**Pull request by asifsmohammed**
Signed-off-by: Asif Sohail Mohammed <nsifmoh@amazon.com>

### Description
Added Data Prepper 1.4.0 linux archives, Dockerfile image and updated version to 1.4.0

Please wait for confirmation before merging.
 
### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Comment by asifsmohammed** [comment]
Please wait for confirmation before merging. Our release date for 1.4.0 is 05/17/2022. We should be ready by EOD.

**Review by dlvenable — APPROVED** [review]

**Review by krisfreedain — APPROVED** [review]
Looks good

**Pull request merged**
